package.json

```diff
@@ -11,11 +11,12 @@
   ],
   "scripts": {
     "postinstall": "bower cache clean && bower --allow-root install",
-    "clean": "rimraf static/dist",
+    "clean": "rm -rf static/dist/*",
```

Comment:
Why do you replace `rimraf` with `rm -rf`? It is not a priority but it breaks the build on Windows OS.

Reply:
This was one of many issue when running those tests on docker. `static/dist` was a mounted volume and couldn't be removed. You're right about breaking compatibility, I didn't think about that. I'll try to solve that using `rimraf`